meetings/2022-12-14.md

```diff
@@ -0,0 +1,80 @@
+
+### Meeting Guidelines
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+* SC: This is a special topic meeting focusing on reviewing the existing material, and identifying areas to improve towards the development and integration of Use Cases.
+
+
+### Process
+* SC: https://github.com/solid/process/ describes stakeholders, documenting use cases, and encouraging adherence to the W3C Web Platform Design Principles' Priority of Constituencies: https://www.w3.org/TR/design-principles/#priority-of-constituencies .
+
+
+### Technical Reports
+* SC: User stories: https://github.com/solid/user-stories/
+* SC: Call to deliver Use Cases and Requirements (UCR) documents alongside Technical Reports (TR):
+  * https://github.com/solid/specification/issues/9 
+* SC: Current UCRs:
+  * https://solid.github.io/authorization-panel/authorization-ucr/
+  * https://solid.github.io/notifications-panel/notifications-ucr/
+  * https://github.com/solid/webid-profile/issues/55 ( https://github.com/solid/webid-profile/pull/80 )
+* SC: Involving the DEIT (Diversity, Equity, Inclusion Team) to review UCs:
+  * https://github.com/solid/deit/blob/main/meetings/2021-11-09.md#deit-to-review-use-cases-in-specification-development
+
+
+### Documentation and Discovery
+* SC: We have a lot of information and so there is room to improve UCs discovery and communication.
+* SC: Questions/Concerns on how to improve:
+
+
+### Contribution Guidelines
+* SC: How to contribute to user stories/cases (documentation and in meetings):
+  * https://github.com/solid/specification/blob/main/CONTRIBUTING.md
+* SC: Describes participants proposing topics before or during meeting:
+  * https://github.com/solid/specification/blob/main/meetings/README.md
+* SC: Topic proposals following the meeting template:
+  * https://github.com/solid/specification/blob/main/meetings/template.md
+
+
```

Comment:
### Polls or Surveys
* VB: We can also look into ways to gather data about Use Cases or feedback about process, among other things.
* VB: There is WIP on [Solid Community Survey](https://virginiabalseiro.github.io/survey) that will be published on solidproject.org.
* VB: Can we use Solid apps to collect and share this data? Which other tools can we use that's in the spirit of the Solid project (open source, good privacy / terms of service).